debug.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import eko
```

Comment:
Do you actually want to merge this?
(might be fine, but maybe I'd not put top level)

In case you used for testing correctness of `pineko`, consider converting in a test (if makes use of `lhapdf` in an "isolated benchmark")

Reply:
Actually I did not commit this (and not even coded). Maybe it is from @felixhekhorn

Reply:
I will resolve this